scripts/validate_container.sh

```diff
@@ -137,7 +137,7 @@ echo "============================="
 
 # Note: piping the `find` results through `sort` guarantees repeatable ordering.
 tmpFile=$(mktemp)
-for ex in `find examples/ applications/ targets/ -name '*.cpp' | sort`;
+for ex in `find examples/ applications/ targets/ -name '*.cpp' -not -path "*/dynamics/*" | sort`;
```

Comment:
Why are we excluding the dynamics examples etc. from the validation? 
Where are these validated?

Reply:
1) anything that requires gpus is not validated during CI or deployment, only during publishing
2) publishing (Validate cuda-quantum image) is exactly calling into this script to do the validation. Hence, disabling it in this script disables that validation. I am not seeing these being validated anywhere. What am I missing?